internal/webhooks/machinedeployment.go

```diff
@@ -38,6 +38,7 @@ import (
 
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 	"sigs.k8s.io/cluster-api/feature"
+	names "sigs.k8s.io/cluster-api/internal/topology/names"
```

Comment:
should this be topologynames for consisentcy (+ eventually add a rule in https://github.com/kubernetes-sigs/cluster-api/blob/7a7ba0110f110003f9ee0d2dcc99988ffb666d54/.golangci.yml#L171